files/build_templates/sflow.service.j2

```diff
@@ -14,6 +14,3 @@ ExecStartPre=/usr/bin/{{docker_container_name}}.sh start
 ExecStart=/usr/bin/{{docker_container_name}}.sh wait
 ExecStop=/usr/bin/{{docker_container_name}}.sh stop
 RestartSec=30
-
```

Comment:
Any specific reason for removing this ?

Reply:
In order for the daemon to be managed by hostcfgd, it needs to be removed from sonic.target. If we don't remove, config reload and reboot will implicitly start sflow since it is associated with sonic.target. Removing this means hostcfgd will listen to PortInitDone and only then spawn the daemon